assets/src/application/ui/layout/entityList/filterBar/buttons/EntityListViewButtonGroup.tsx

```diff
@@ -15,7 +15,7 @@ const EntityListViewButtonGroup: React.FC<EntityListViewButtonGroupProps> = ({
 	view,
 }) => {
 	return (
-		<ButtonGroup>
+		<ButtonGroup buttonSize={ButtonSize.SMALL}>
```

Comment:
oh i see... you're passing additional props through to the children...

ya ok so cloning is the way to go... I thought initially it was just being done to add that `--no-border-radius` modifier...

did you see the discussion Manzoor and I had yesterday about different methods for adding review feedback?
here's a perfect example of where I didn't realize why something was done a particular way until after already providing feedback.
normally I would go back and delete those irrelevant comments, but I'll let them stand this time just for fun 😆

Reply:
yes, let it leave here for fun
***
`cloneElement` this is an interesting pattern, the only thing I don't like is that it implies a bit of magic, because prop is passed less explicit